### PR TITLE
Chore/blog holiday hackdays

### PR DESCRIPTION
**Pull request by thorwebdev**
https://zone-www-dot-com-git-chore-blog-holiday-hackdays-supabase.vercel.app/blog/2021/12/03/supabase-holiday-hackdays-hackathon

### CODE REVIEW

File: www/_blog/2021-12-03-supabase-holiday-hackdays-hackathon.mdx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+image: launch-week-three/holiday-hackdays-og.png
+thumb: launch-week-three/holiday-hackdays-og.png
+tags:
+  - hackathon
+  - community
+date: '2021-12-03'
+toc_depth: 3
+---
+
+While not everyone will be celebrating the holidays this December, we can all be united in celebrating open-source software, and to do so we're kicking off the "Holiday Hackdays" (loosely inspired by the [12 Pubs of Christmas](https://www.irelandbeforeyoudie.com/12-pubs-of-christmas-rules-tips-everything-you-need-for-a-great-night/) only a bit healthier).
```
Comment (kiwicopple) [inline www/_blog/2021-12-03-supabase-holiday-hackdays-hackathon.mdx:17]:
```suggestion
While not everyone will be celebrating the holidays this December, we can all be united in celebrating open-source software, and to do so we're kicking off the "Holiday Hackdays" (loosely inspired by the [12 Pubs of Christmas](https://www.irelandbeforeyoudie.com/12-pubs-of-christmas-rules-tips-everything-you-need-for-a-great-night/), only a bit healthier).
```

File: www/_blog/2021-12-03-supabase-holiday-hackdays-hackathon.mdx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+[As always](https://supabase.com/blog/2021/10/14/hacktoberfest-hackathon-winners-2021#the-prizes), we'll have some extremely limited edition swag to give away that can only be won by participating in the Hackathon!
+
+The Supabase team will be participating as well, for example [Jon](https://twitter.com/_dijonmusters) and I, inspired by [Egghead's "Holiday Course Release Extravaganza"](https://egghead.io/20-days), will be building an Advent Calendar for all the [awesome community content](https://supabase.com/blog/2021/11/29/community-day#new-tutorials-and-integration-guides) you've been creating. Get ready for a cool open-source project which will be using a lot of the hottest Supabase features! To make sure you don't miss this one, follow us on [Twitter](https://twitter.com/supabase) and subscribe to our [YouTube channel](https://www.youtube.com/supabase). We'll see you there.
+
+## Details
+
+### Timeline
+
+- **Friday Dec 3rd at 07:00am PT:** Last day of Launch Week. Join us on [Twitter Spaces](https://twitter.com/i/spaces/1eaKbNDPEOYKX?s=20) for the kickoff announcement.
```
Comment (kiwicopple) [inline www/_blog/2021-12-03-supabase-holiday-hackdays-hackathon.mdx:29]: is this the correct link for spaces?

File: www/_blog/2021-12-03-supabase-holiday-hackdays-hackathon.mdx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+- Build your project during the next 10 days and hang out with the community [on Discord](https://discord.gg/bnncdqgBSS).
+- **Sunday Dec 12th at 11:59pm PT:** Submission deadline
+- Judges Deliberate (Monday)
```
Comment (kiwicopple) [inline www/_blog/2021-12-03-supabase-holiday-hackdays-hackathon.mdx:32]: should we indent these points that happen after the key days? up to you

File: www/_blog/2021-12-03-supabase-holiday-hackdays-hackathon.mdx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+<blockquote class="twitter-tweet" data-theme="dark"><p lang="en" dir="ltr">Hacktoberfest Hackathon Swag finally arrived! 🥇🥈🥨 see the winners here: <a href="https://t.co/wgRBu92Bq4">https://t.co/wgRBu92Bq4</a> <a href="https://t.co/i1eD1pdA8I">pic.twitter.com/i1eD1pdA8I</a></p>&mdash; Supabase (@supabase) <a href="https://twitter.com/supabase/status/1463355998928736262?ref_src=twsrc%5Etfw">November 24, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+### Prize categories
+
+There are multiple chances to win, there will be prizes for:
+
+- Most impressive usage of the Realtime API features
+- Most fun / holiday themed project 
+
+There will be a winner and a runner-up prize for each category.
```
Comment (kiwicopple) [inline www/_blog/2021-12-03-supabase-holiday-hackdays-hackathon.mdx:44]: add an image of the $1M gold t shirt?

**Review by kiwicopple — APPROVED** [review]

File: www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx [hunk 5]
```diff
@@ -57,7 +57,7 @@ We've released [Supabase Studio](https://github.com/supabase/supabase/tree/maste
 
 Read the [blog post](/blog/2021/11/30/supabase-studio). 
 
-Tune in on [Twitter Spaces](https://twitter.com/i/spaces/1yNGaYyyRPRGj) at 7am PT!
+Watch the Twitter Spaces recap on [YouTube](https://youtu.be/9zmaKt6evxw).
```
Comment (kiwicopple) [inline www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx:60]: fix link

File: www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx [hunk 6]
```diff
@@ -69,11 +69,13 @@ Wednesday 1 December 2021: 7am PT
...
 
-Don't let the grinch steal your data this Christmas - we have a huge security update coming for Realtime.
+We released Realtime Row Level Security.
 
 ### Where?
 
-Blog post coming soon. Follow our [Twitter](https://twitter.com/supabase) or [Discord](https://discord.supabase.com) to get notified immediately.
+Read the [blog post](/blog/2021/12/01/realtime-row-level-security-in-postgresql). 
+
+Watch the Twitter Spaces recap on [YouTube](https://youtu.be/9zmaKt6evxw).
```
Comment (kiwicopple) [inline www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx:78]: fix link

File: www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx [hunk 7]
```diff
@@ -85,11 +87,13 @@ Thursday 2 December 2021.
...
 
-What's in the box? This one's a secret so you'll just need to dial in on Thursday.
+We acquired [Logflare](https://logflare.app).
 
 ### Where?
 
-Blog post coming soon. Follow our [Twitter](https://twitter.com/supabase) or [Discord](https://discord.supabase.com) to get notified immediately.
+Read the [blog post](/blog/2021/12/02/supabase-acquires-logflare). 
+
+Watch the Twitter Spaces recap on [YouTube](https://youtu.be/9zmaKt6evxw).
```
Comment (kiwicopple) [inline www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx:96]: fix link

File: www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx [hunk 8]
```diff
@@ -101,12 +105,9 @@ Friday 3 December 2021.
...
 
-Let's be honest. It's never just one more thing.
+Let's be honest. It's never just one more thing. Today we announced five more things 🔥.
 
 ### Where?
 
-Blog post coming soon. Follow our [Twitter](https://twitter.com/supabase) or [Discord](https://discord.supabase.com) to get notified immediately.
-
-## Build in public
+Read the [blog post](/blog/2021/11/30/supabase-studio).
```
Comment (kiwicopple) [inline www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx:112]: link to blog post

**Review by kiwicopple — APPROVED** [review]

File: www/_blog/2021-12-03-supabase-holiday-hackdays-hackathon.mdx (hunk 4 above)
Comment (thorwebdev) [reply www/_blog/2021-12-03-supabase-holiday-hackdays-hackathon.mdx:44]: we have the supabase hacktoberfest swag tweet instead

File: www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx (hunk 5 above)
Comment (kiwicopple) [reply www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx:60]:
```suggestion
Watch the Twitter Spaces recap on [YouTube](https://youtu.be/n1QGq75ZzvE).
```

File: www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx (hunk 6 above)
Comment (kiwicopple) [reply www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx:78]:
```suggestion
Watch the Twitter Spaces recap on [YouTube](https://youtu.be/4thVj8j19s).
```

File: www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx (hunk 7 above)
Comment (kiwicopple) [reply www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx:96]:
```suggestion
Watch the Twitter Spaces recap on [YouTube](https://youtu.be/4thVj8j19s).
```

File: www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx (hunk 8 above)
Comment (kiwicopple) [reply www/_blog/2021-11-26-supabase-launch-week-the-trilogy.mdx:112]:
```suggestion
Read the [blog post](/blog/2021/12/03/launch-week-three-friday-five-more-things). 
```

File: www/_blog/2021-12-03-launch-week-three-friday-five-more-things.mdx [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+authorURL: https://github.com/awalias
+image: launch-week-three/five-more-things/five-more-things-og.png
+thumb: launch-week-three/five-more-things/five-more-things-thumb.png
+tags:
+  - launch-week
+  - graphql
+  - CDN
+  - storage
+  - functions
+date: '2021-11-26'
```
Comment (kiwicopple) [inline www/_blog/2021-12-03-launch-week-three-friday-five-more-things.mdx:16]:
```suggestion
date: '2021-12-03'
```

**Pull request merged**